### PR TITLE
Allow logius updates to come through

### PR DESCRIPTION
**Pull request by joeribekker**
Fixes several flow issues, most notibly that the flow doesn't crash when an object already exists.

* Add uniques constraints to Logius.  Also changed an O2O to a FK for themes and informationareas.
* Allow tasks to be performed in the admin
* Update Logius entries if they already exists (instead of crashing)
* Added filters for UPNs in admin.
* If form has no Meta, don't check for exclude

I'm not really happy that I had to introduce a unique together on UPN URI and grondslag, but it seems some have the same URI meant for a different layer of government. I'm not sure how to treat these but adding this unique constraint seems the most appropriate to prevent overwrites based on just the UPN URI.

### CODE REVIEW

File: src/sdg/core/apps.py [hunk 1]
```diff
@@ -37,3 +37,6 @@ class CoreConfig(AppConfig):
     def ready(self):
         post_migrate.connect(register_project_models, sender=self)
         post_migrate.connect(load_app_fixtures, sender=self)
+
+        # This makes the tasks available in the Django Admin.
+        from . import tasks
```
Comment (dariomory) [inline src/sdg/core/apps.py:42]: Can you clarify why this is needed? The comment may not be specific, as it already shows in the admin without it.

File: src/sdg/core/models/logius.py [hunk 2]
```diff
@@ -189,6 +190,7 @@ def generate_initial_data(self, catalog: ProductenCatalogus):
     class Meta:
         verbose_name = _("uniforme productnaam")
         verbose_name_plural = _("uniforme productnamen")
+        unique_together = (("upn_uri", "grondslag"),)
```
Comment (dariomory) [inline src/sdg/core/models/logius.py:193]: Here it's better use the constraints option instead:

```
class Meta:
        constraints = [
            models.UniqueConstraint(fields=["upn_uri", "volume_number"], name="unique_uri_and_grondslag")
        ]
```

Ref: https://docs.djangoproject.com/en/3.2/ref/models/options/#unique-together

File: src/sdg/core/management/utils.py [hunk 3]
```diff
@@ -57,30 +57,36 @@ def load_informatiegebieden(data: List[Dict[str, Any]]) -> int:
     codes = [obj.pop("SDG_Code") for obj in data]
     grouped_data = dict(zip(codes, data))
 
-    informatiegebied_list = [
-        Informatiegebied(
+    count_informatiegebieden = 0
```
Comment (dariomory) [inline src/sdg/core/management/utils.py:60]: This  variable is unused. Probably better to either remove it or add it to the returned count.

**Review by dariomory — CHANGES_REQUESTED** [review]
Good one :) Added some observations.

File: src/sdg/core/apps.py (hunk 1 above)
Comment (joeribekker) [reply src/sdg/core/apps.py:42]: Ah, I was looking for that import but could not find it.

File: src/sdg/core/models/logius.py (hunk 2 above)
Comment (joeribekker) [reply src/sdg/core/models/logius.py:193]: Altho I prefer readability from the "old way", the threat of deprecation is indeed reason to use the new way.

**Pull request merged**